ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/store/serialization/KeyHasher.scala

```diff
@@ -79,6 +79,13 @@ object KeyHasher extends KeyHasher {
         val z1 = op(z, HashTokenCollectionBegin(0))
         op(z1, HashTokenCollectionEnd())
 
+      // Variant: [CollectionBegin(), Text(tag), Token(value), CollectionEnd()]
+      case ValueBuiltinException(tag, v) =>
+        val z1 = op(z, HashTokenCollectionBegin(1))
+        val z2 = op(z1, HashTokenText(tag))
+        val z3 = foldLeft(v, z2, op)
+        op(z3, HashTokenCollectionEnd())
+
       // Variant: [CollectionBegin(), Text(variant), Token(value), CollectionEnd()]
```

Comment:
code outside daml-lf, please make this one crash.

Reply:
ok
I left the code, but commented out, because it seems correct to me